### PR TITLE
feat: User Availability state

### PR DESCRIPTION
**Pull request by przemvs**
## Description

New User Availability State is displayed on Avatar instead of before user name. Colors are changed to proper status color.
We don't want to display availability state on big User avatars, right sidebar and also in call.

## Screenshots/Screencast (for UI changes)

image

image

image

## Checklist

- [ ] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

### Important details for the reviewers

(Delete this section if unnecessary)

- use (x) data
- can be reviewed commit-by-commit
- be sure to look at ...

### CODE REVIEW

File: src/script/components/calling/CallParticipantsListItem/CallParticipantItemContent/CallParticipantItemContent.tsx [hunk 1]
```diff
@@ -60,12 +57,7 @@ export const CallParticipantItemContent = ({
     <div css={wrapper}>
       <div css={contentText}>
         <div css={nameWrapper(isAudioEstablished)}>
-          <UserInfo
-            user={user}
-            css={[userAvailability, ellipsis]}
-            dataUieName="status-name"
-            showAvailability={selfInTeam}
-          />
+          <UserInfo user={user} css={[ellipsis]} />
```
Comment (PatrykBuniX) [inline src/script/components/calling/CallParticipantsListItem/CallParticipantItemContent/CallParticipantItemContent.tsx]: question: We don't need "status-name" data-uie-name param anymore?

Comment (przemvs) [reply src/script/components/calling/CallParticipantsListItem/CallParticipantItemContent/CallParticipantItemContent.tsx]: We need, probably accidentally remove. Addressed in: https://github.com/wireapp/wire-webapp/pull/17304/commits/04a1969b92e4a1371c894d278ad6663f12478c0b

File: src/script/components/list/ConversationListCell.tsx [hunk 2]
```diff
@@ -185,22 +183,12 @@ const ConversationListCell = ({
...
+            {!isGroup && !!users.length && <Avatar participant={users[0]} avatarSize={AVATAR_SIZE.SMALL} />}
           </div>
 
           <div className="conversation-list-cell-center">
             {is1to1 ? (
-              <UserInfo
-                className="conversation-list-cell-availability"
-                user={conversation.firstUserEntity()!}
-                theme={isActive}
-                dataUieName="status-availability-item"
```
Comment (przemvs) [inline src/script/components/list/ConversationListCell.tsx:201]: This one is unnecessary now. We don't have status-availability in UserInfo

**Review by PatrykBuniX — APPROVED** [review]
Looks good to me

**Review by atomrc — APPROVED** [review]
Nice cleaning of the `.less` files 🙏  Nice job man 😎

**Review by V-Gira — APPROVED** [review]

**Pull request merged**